sql/2019/12_Mobile_Web/12_18.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+
+# <link rel="manifest">
```

Comment:
Should we use the `installable-manifest` Lighthouse audit? It seems that one is more rigorous than checking whether the page simply _has_ a manifest, it also verifies that it's installable. Docs: https://developers.google.com/web/tools/lighthouse/audits/install-prompt